### ISSUE TITLE
unwrapped vars wont render

### ISSUE THREAD

**Issue by titoBouzout** [ryansolid/mobx-jsx#20]
I love mobx and the concept of an almost featureless rendering that only changes what actually changes, Im planning to use this. I found just one problem, in a render function vars should be wrapped, the following will work:

```
return (
	<div>
		<span>{this.counter}</span>
		<Test counter={100} /> . <Test counter={200} />
	</div>
)
```

but the following doesnt work:

```
return (
	<div>
		{this.counter}
		<Test counter={100} /> . <Test counter={200} />
	</div>
)
```

Is there any chance to do autowrapping or something to avoid this problem, and thanks!

**Comment by ryansolid** [comment]
Thanks for reporting. This was working before. I made some updates in the last version cross libraries and it looks like I missed updating the function signature in this library. I will fix it tonight.

**Comment by ryansolid** [comment]
Ok fixed in 0.11.6. This was a fairly widespread bug that would have affected any element with more than a single dynamic child. Thanks for reporting. I'm sure others hit this and just didn't report it.

**Comment by titoBouzout** [comment]
Yes, at first it looked more like a limitation than a bug. I really want to use this that's why I reported it, I just tested it and it works, thank you!
